src/README.md

```diff
@@ -88,26 +115,26 @@ The DAOS stack is organized as a set of micro-services over a client/server arch
 - <a href="client/dfs/README.md">POSIX File & Direcotory Emulation</a>
 - <a href="common/README.md">Common Library</a>
 
-<a id="300"></a>
+<a id="4"></a>
 ## Software Compatibility
 
-Interoperability in DAOS is handled via protocol and schema versioning for PM data structures.
+Interoperability in DAOS is handled via protocol and schema versioning for persistent data structures.
 
-<a id="301"></a>
+<a id="41"></a>
 ### Protocol Compatibility
 
 Limited protocol interoperability is to be provided by the DAOS storage stack. Version compatibility checks will be performed to verify that:
 
 * All targets in the same pool run the same protocol version.
 * Client libraries linked with the application may be up to one protocol version older than the targets.
 
-If a protocol version mismatch is detected among storage targets in the same pool, the entire pool will fail to start up and will report failure to the control API. Similarly, connection from clients running a protocol version incompatible with the targets will return an error.
+If a protocol version mismatch is detected among storage targets in the same pool, the entire DAOS system will fail to start up and will report failure to the control API. Similarly, connection from clients running a protocol version incompatible with the targets will return an error.
 
-<a id="302"></a>
+<a id="42"></a>
 ### PM Schema Compatibility and Upgrade
 
-The schema of persistent data structures might evolve from time to time to fix bugs, add new optimizations or support new features. To that end, VOS supports schema versioning.
+The schema of persistent data structures might evolve from time to time to fix bugs, add new optimizations or support new features. To that end, ther persistent data structures support schema versioning.
```

Comment:
sp - "ther" -> "the"

Reply:
I prefer may over might since its formally defined in RFC2119.